core/src/main/resources/xml/xsl/stub4testtool.xsl

```diff
@@ -352,8 +352,8 @@
 		</xsl:element>
 	</xsl:template>
 	
-	<xsl:template match="param/@pattern[contains(.,'{now,')]">
-		<xsl:attribute name="pattern"><xsl:value-of select="replace(.,'\{now,','{fixedDate,')"/></xsl:attribute>
+	<xsl:template match="param/@pattern[contains(.,'{now')]">
```

Comment:
Dit matcht ook op '{nowadays}'. Is dat niet erg? (zou best kunnen)

Reply:
Dat is wel erg, want pattern mag ook een session key of andere parameter zijn, dus als iemand dit zou doen met {nowadays} als parameter zou dit inderdaad fout gaan.